kubernetes-model-generator/generateModelDocker.sh

```diff
@@ -18,13 +18,13 @@
 set -e
 
 BASEDIR=$(dirname "$BASH_SOURCE")
-ABSOLUTE_BASEDIR=$(realpath "$BASEDIR")
+ABSOLUTE_BASEDIR=$(realpath "$BASEDIR/..")
 
-DOCKER_IMAGE_GOLANG=golang:1.15.6
+DOCKER_IMAGE_GOLANG=marcnuri/golang-1.15-java11
```

Comment:
Oooh, that looks like a maintenance nightmare!

Reply:
:)

https://github.com/manusa/docker-images/blob/2dab5af4baef3616cc7ad02ba7c8d27addb32aa9/golang-1.15-java11/Dockerfile#L8

Not really, it's a very simple image, but I haven't found any other, open, publicly maintained equivalent (I bet not many Golang developers want to mix it with Java, and vice-versa)